src/components/Thought.js

```diff
@@ -437,7 +469,7 @@ const ThoughtContainer = ({
     'cursor-parent': isCursorParent,
     'cursor-grandparent': isCursorGrandparent,
     'code-view': isCodeView,
-    dragging: isDragging,
+    dragging: (highlightBullet && equalThoughtsRanked(draggedThoughtsRanked, thoughtsRanked)) || isDragging,
```

Comment:
I recommend putting `isDragging` first. Putting pre-calculated values first in a boolean expression allows it to short-circuit and avoid calculating the full expression when it doesn't have to.